etc/ReplayOfflineConfiguration.py

```diff
@@ -587,13 +587,26 @@
                do_reco=True,
                scenario=ppScenario)
 
-DATASETS = ["ParkingSingleMuon","ParkingSingleMuon0","ParkingSingleMuon1","ParkingSingleMuon2"]
+DATASETS = ["ParkingSingleMuon1","ParkingSingleMuon2","ParkingSingleMuon3",
```

Comment:
This does not seem to have `ParkingSingleMuon0` (while the HLT menu does have it).